frontend/js/integrations/search.js

```diff
@@ -159,6 +159,7 @@ define([
                         RegExp.prototype.test.bind(/application\/.*|video\/.*/),
                         _.property("mimetype")
                     ));
+                videos = _.sortBy(videos, "master").reverse();
```

Comment:
Can we integrate this in the `sort` below? Something like

```js
videos.sort(
    util.lexicographical([
        util.firstWith(
            _.property("master")
        ),
        // [...]
    ])
);
```

Should work, but please double-check me on this.